packages/blade-old/MIGRATION.md

```diff
@@ -0,0 +1,166 @@
+- [From version 1.4.x to 2.x.x](#from-version-14x-to-2xx)
+
+
+# From version 1.4.x to 2.x.x
+
+## Migration guide for new blade-old icons
+
+### Codemod to migrate to new API
+
+We've written a codemod to automate the process of replacing the `icon` prop with imported icons. 
+
+Simply install the blade-old package and run
+
+```sh
+// migrate-icons [directory_path_glob] [...jscodeshift_options]
+migrate-icons ./src/App/*
+```
+
+This codemod automatically changes 
+
+This:
+
+```jsx
+<Icon size="md" name="download" {...{}}>
+    <works />
+</Icon>
+```
+
+To this:
+
+```jsx
+import { Download } from "@razorpay/blade-old";
+
+<Download size="md" {...{}}>
+    <works />
+</Download>
+```
+
+### Limitations & Edge cases of codemod
+
```

Comment:
add
There are some edge cases where its codemod won't work and hence you need to do those changes *manually*